pandas/core/window/rolling.py

```diff
@@ -442,6 +444,7 @@ def _apply(
         floor: int = 1,
         is_weighted: bool = False,
         name: Optional[str] = None,
+        use_numba_cache: Optional[bool] = False,
```

Comment:
is there a reason we would not want this ever? (e.g. shouldn't we *always* cache)

Reply:
The variable name is a bit of a misnomer. `use_numba_cache` tells a downstream routine (`_apply`) that we _can_ cache the numba function

i.e. this is `False` for every other rolling method that cant cache yet and can be `True` for apply

Reply:
k cool, can you document that somewhere

Reply:
This can just be annotated as `bool` no? Or do we need to explicitly handle `None`?

Reply:
seconding Will's question.  this appears above too